### PR TITLE
Updated so checks are passing

### LINKED ISSUE #3: Make documentation more clear that this library only runs on CPython

### PR DESCRIPTION
**Pull request by makermelissa**
Also fixes #3. Updated pre-commit, ran precommit, linted, fixed similarity check, made more clear this is for CPython.

### CODE REVIEW

**Comment by makermelissa** [comment]
Could I please get a review on this? It has been a week. Thanks.

File: .pylintrc [hunk 1]
```diff
@@ -22,8 +22,7 @@ ignore-patterns=
 #init-hook=
 
 # Use multiple processes to speed up Pylint.
-# jobs=1
-jobs=2
+jobs=1
```
Comment (dhalbert) [inline .pylintrc]: Is it no longer true you can use `jobs=2`? If so could you update the comment to explain why?

**Review by dhalbert — CHANGES_REQUESTED** [review]
Thanks! Just one thing re building.

File: .pylintrc (hunk 1 above)
Comment (makermelissa) [reply .pylintrc]: Yeah, it caused a race condition which ignored the `min-similarity-lines` setting.

**Review by dhalbert — APPROVED** [review]

**Pull request merged**